frame/staking/src/pallet/impls.rs

```diff
@@ -727,6 +729,7 @@ impl<T: Config> Pallet<T> {
 		let mut nominators_taken = 0u32;
 
 		let mut sorted_voters = T::VoterList::iter();
+		let mut min_active_bond = None;
```

Comment:
I have not seen (or like) using option like this. Start with `Bounded::max_value()` and go downward.